examples/nvidia_dali/dali_config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "batch_size" : 5,
+  "num_threads" : 2,
+  "device_id" : 0
```

Comment:
Does these parameter work?

Reply:
Yes by default they hold negative values and will throw errors like `RuntimeError: Device index must not be negative`